ai2thor_wrapper/envs.py

```diff
@@ -0,0 +1,195 @@
+        # Loads config file from path relative to ai2thor_wrapper python package folder
+        config_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), config_path)
+        self.config = configparser.ConfigParser()
+        config_output = self.config.read(config_path)
+        if len(config_output) == 0:
+            print('No config file found at: {}. Exiting'.format(config_path))
+            sys.exit()
+
+        self.scene_id = scene_id
+        self.controller = ai2thor.controller.Controller()
+        self.controller.start()
+        if seed:
+            self.seed(seed)
+
+        self.controller.reset(self.scene_id)
+        self.event = self.controller.step(dict(action='Initialize', gridSize=0.25, renderDepthImage=True,
+                                               renderClassImage=True,
+                                               renderObjectImage=True))
+
+        self.current_object_type = current_object_type
+        self.max_episode_length = max_episode_length
+        self.t = 0
+        self.done = False
+        self.task = task
+        self.movement_reward = movement_reward
+
+        # action dictionary from int to dict action to pass to event.controller.step()
+        self.ACTION_SPACE = {}
+        for idx, action_str in enumerate(POSSIBLE_ACTIONS):
+            self.ACTION_SPACE[idx] = dict(action=action_str)
+
+        if not interaction:
+            interaction_actions = ['OpenObject', 'CloseObject', 'PickupObject', 'PutObject']
+            for idx in range(len(POSSIBLE_ACTIONS)):
+                if self.ACTION_SPACE[idx]['action'] in interaction_actions:
+                    self.ACTION_SPACE.pop(idx)
+
+        self.NUM_ACTIONS = len(self.ACTION_SPACE.keys())
+        self.action_space = self.NUM_ACTIONS
+
+        # acceptable objects taken from config.ini file. Stripping to allow spaces
+        self.pickup_object_types = [x.strip() for x in self.config['ENV_SPECIFIC']['PICKUP_OBJECTS'].split(',')]
+        self.receptacle_object_types = [x.strip() for x in self.config['ENV_SPECIFIC']['ACCEPTABLE_RECEPTACLES'].split(',')]
+        print('Objects that can be picked up: \n{}'.format(self.pickup_object_types))
+        print('Objects that allow objects placed into/onto them (receptacles): \n{}'.format(self.receptacle_object_types))
+
+        self.grayscale = grayscale
+        self.resolution = (128, 128)  # (64, 64)
+        if self.grayscale:
+            self.observation_space = np.array((1,) + self.resolution)
+        else:
+            self.observation_space = np.array((3,) + self.resolution)
+
+        self.goal_objects_collected_and_placed = []
+        self.last_amount_of_goal_objects = len(self.goal_objects_collected_and_placed)
+
+    def step(self, action_int):
+        if self.ACTION_SPACE[action_int]['action'] == 'PickupObject':
+            if len(self.event.metadata['inventoryObjects']) == 0:
+                for o in self.event.metadata['objects']:
```

Comment:
Give me at least 3 letters. I hate having one, I forget what the name means
```suggestion
                for obj in self.event.metadata['objects']:
```